### DISCUSSION TITLE
Can't reconfigure via Loupedeck software

### DISCUSSION THREAD

**Discussion by macbrush** [calibx/msfsdeck#223] [Q&A]
Hi, thanks for a great plugin, I enjoy using it with my Loupedeck CT very much. But I have an issue, I can't reconfigure button and dial assignments beyond the first page, when I hit a button to go to another page (1,2,3... etc), say to AP page, the hardware does switch to AP page, however the software doesn't follow like with other plugins, hence I am stuck with reconfiguring the first page only. Please advice. Thank you very much.

**Comment by RubiXKube** [comment]
after adding a new pages to your profile, you have to set this page to button
![Sans titre](https://user-images.githubusercontent.com/66200975/228410917-21e15182-8e25-4642-9205-cb9071971cde.png)

**Reply by cfought76** [reply]
That doesn't answer the question. That is fine if you want to assign a page to a button, but the question is how do we edit a page? It doesn't appear in the software to edit.

**Comment by calibx** [comment]
Hi, the "folders" are not configurable (AP, Airliner, Lights ...) by the UI => they come with the plugin.
But you can make your own pages drag/dropping items.
